tutorials/pavics/pavics-intro.md

```diff
@@ -0,0 +1,6 @@
+# PAVICS Documentation Examples
+
+## Example Notebooks:
+
```

Comment:
Change secondary title to "Table of Contents" or something that doesn't repeat the word "examples"